manifests/kustomize/env/platform-agnostic-kubernetes/kustomization.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../base/crds
```

Comment:
seems like the crds should be in /manifests/kustomize/base/pipeline/cluster-scoped, not base/crds

assuming we always want to deploy these?

unless we don't in which case maybe it makes more sense to couple these with the env folder

Reply:
Since K8s Native API is an opt-in feature, moving to cluster-scoped path will always deploy manifests that might not be used by the user.